app/views/layouts/application.html.erb

```diff
@@ -17,11 +17,6 @@
       <meta name="shared_session_key" content="<%= shared_session_key %>">
     <%- end %>
 
-    <script>
-      window.EmberENV = window.EmberENV || {};
-      window.EmberENV['FORCE_JQUERY'] = true;
-    </script>
```

Comment:
This is already in `env.js` and included in `ember_query.js`

https://github.com/discourse/discourse/blob/ce6a0a5e9e33e3cf7459c180eb566e5f12ec1371/app/assets/javascripts/ember_jquery.js#L1